### ISSUE TITLE
New version(0.1.36) missing fixtures atribute in ServerConfig

### ISSUE THREAD

**Issue by eini-rc** [miragejs/miragejs#392]
```
export interface ServerConfig {
    urlPrefix?: string;
    fixtures?:any; // -> this line missing
    ...
  }
```
untill previous version we used:
```
export default new Server({
  environment: '',
  models: {
     ...
  },
  fixtures: {
    ....
  },
  seeds(server) {
    server.loadFixtures()
  },
  routes() {
   ...
  }
})
```
but after update to v: 0.1.36 `fixtures` is not parameter of `ServerConfig`

**Comment by samselikoff** [comment]
@zoltan-nz any ideas here?

**Comment by zoltan-nz** [comment]
@samselikoff @eini-rc Good call. 

It was not added to the interface because it isn't exist in the official API. https://miragejs.com/api/classes/server/

@samselikoff please help, where we can find the real source of truth?

**Comment by zoltan-nz** [comment]
It looks to me that this line is the only reference: https://github.com/miragejs/miragejs/blob/master/lib/server.js#L689

So `fixtures` can be part of the ServerConfig, maybe we should update the API doc as well.

**Comment by samselikoff** [comment]
Yes, I've been meaning to add it actually, there's a bug issue for it here https://github.com/miragejs/miragejs/issues/324

Will definitely make its way into API eventually but yes all the config options are public 👍

**Comment by zoltan-nz** [comment]
@samselikoff A PR is opened, so `fixtures` gonna be part of the `ServerConfig` after that. ;)

**Comment by samselikoff** [comment]
@eini-rc released in 0.1.37 https://github.com/miragejs/miragejs/releases/tag/v0.1.37

**Comment by eini-rc** [comment]
@samselikoff @zoltan-nz Thanks for the quick fix!!!
